pkg/operator/ceph/controller/spec.go

```diff
@@ -57,6 +57,8 @@ const (
 	CephUserID                              = 167
 	livenessProbeTimeoutSeconds       int32 = 5
 	livenessProbeInitialDelaySeconds  int32 = 10
+	readinessProbePeriod              int32 = 5
+	readinessProbeInitialDelaySeconds int32 = 5
```

Comment:
```suggestion
	readinessProbeInitialDelaySeconds int32 = 5
```
any reason to keep value 5? since we 10 for livenessprobe and also default value is 10 in k8s doc?
I'm asking since we have seen in somecluster where it is very slow and sometime pods dies due to probes.

Reply:
> any reason to keep value 5? since we 10 for livenessprobe and also default value is 10 in k8s doc? I'm asking since we have seen in somecluster where it is very slow and sometime pods dies due to probes.

TBH I just picked a small value. As of my testing it seems to work but obviously I didn't test it on a bigger cluster. Anyway from the k8s docs [1] the default value seem to be 0 and not 10. Reasdiness probe shouldn't lead to killing/restarting the pod and normally the pod wouldn't be considered in failure unless the readiness test fails for `FailureThreshold` consecutive times (defaults to 3). So with the current configuration (5 seconds) in fact in order for k8s to mark the pod as in failure it will need to fail for `InitialDelaySeconds + FailureThreshold*PeriodSeconds = 5 + 3*5 = 20 sec`. WDYT? If you still think 5 is a small value, I can increase it to 10 in that case the new time would be `10 + 3*5 = 25 sec`. 

[1] https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

Reply:
oh right. Thanks for explaining